### PR TITLE
Update copyright headers to include Open Enclave SDK contributors

### PR DESCRIPTION
**Pull request by CodeMonkeyLeet**
Per contribution of the project to Confidential Computing Consortium, update
all the copyright headers on source files to include all Open Enclave SDK
contributors. All copyright headers should now read:

```
Copyright (c) Open Enclave SDK contributors.
Licensed under the MIT License.
```

- Update scripts/check-license to enforce this new value on git commit.

### CODE REVIEW

**Comment by CodeMonkeyLeet** [comment]
Marked as `do not merge` while waiting on legal to determine where the copyright issue will land.

File: scripts/VirtualMachineSgxSettings.psd1 [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 # Licensed under the MIT License.
 
 @{
     GUID              = "{5c36f9d9-e64f-4dbb-ae38-6ad1669954c4}"
-    Author            = "Microsoft Corporation"
-    CompanyName       = "Microsoft Corporation"
-    Copyright         = "© Microsoft Corporation. All rights reserved."
+    Author            = "Open Enclave SDK contributors"
+    CompanyName       = "Confidential Computing Consortium"
+    Copyright         = "© Microsoft Corporation and Open Enclave SDK contributors."
```
Comment (dthaler) [inline scripts/VirtualMachineSgxSettings.psd1]: Need to update this copyright statement

Comment (CodeMonkeyLeet) [reply scripts/VirtualMachineSgxSettings.psd1]: Thanks for the catch, fixed!

**Pull request merged**